<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminQueriesClass</class>
 <widget class="QWidget" name="AdminQueriesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запросы администратора</string>
  </property>
  <widget class="QPushButton" name="metal_price_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Цена всего металла на складе</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="stones_price_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Цена всех камней на складе</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="products_price_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Цена всех изделий</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="upd_metal_price_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Обновить цену всего металла</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="upd_stones_price_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Обновить цену всех камней</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="upd_products_price_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Обновить цену всех изделий</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
